docs/ExceptionsHandling.md

```diff
@@ -0,0 +1,19 @@
+# Exceptions handling
+
+Exceptions handling is an important part of our code. We pay attention to this detail and **we recommend to wrap every statement from this SDK with a `try {} catch() {}` clause**.
+
+You should at least handle `\SumUp\Exceptions\SumUpSDKException` exception but if you want you can handle all sorts of exceptions.
+
+```php
+try {
+    $sumup = new \SumUp\SumUp(/* configuration */);
```

Comment:
Is it possible to use the constructor signature here?

Reply:
I don't understand what you mean